src/filter-by-query.js

```diff
@@ -0,0 +1,13 @@
+/*
+Функция фильтрует элементы массива по заданным значениям
+*/
+
+
+function filterByQuery (array, query) {
+    let Arr = array.filter(function (el) {
```

Comment:
тут можно сразу писать
```suggestion
    return array.filter(function (el) {
```